cypress/integration/comment_box_scpec.js

```diff
@@ -0,0 +1,30 @@
+// describe("Timeline", function() {
```

Comment:
This whole test is commented, do we need to un-comment this test?

Reply:
it has now been un-commented